lib/elixir/lib/module/locals_tracker.ex

```diff
@@ -213,15 +213,21 @@ defmodule Module.LocalsTracker do
 
   defp get_line(meta), do: Keyword.get(meta, :line)
 
+  defp get_position(meta) do
+    {get_line(meta), meta[:column]}
+  end
+
   defp build_meta(nil, _meta), do: []
 
   # We need to transform any file annotation in the function
   # definition into a keep annotation that is used by the
   # error handling system in order to respect line/file.
-  defp build_meta(line, meta) do
-    case Keyword.get(meta, :file) do
-      {file, _} -> [keep: {file, line}]
-      _ -> [line: line]
+  defp build_meta(position, meta) do
+    case {position, Keyword.get(meta, :file)} do
+      {{line, nil}, {file, _}} -> [keep: {file, line}]
+      {{line, nil}, nil} -> [line: line]
+      {{line, col}, {file, _}} -> [keep: {file, line}, column: col]
```

Comment:
Should we be storing this column here? It seems that it indeed refers to the line stored in meta, but later on it will be dropped as we cannot be sure the column refers to the line inside keep.

Maybe we could keep `{line, col}` here but I guess it would be better to do it in another PR

Reply:
You are correct, we should drop it.